test/integration/multi-cluster-deployment/workload-deployment.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e # exit on error
+
+echo "Create a placement to deliver an app to all clusters in wds1."
+echo "This placement configuration determines where to deploy the workload by using the label selector expressions found in clusterSelectors. It also specifies what to deploy through the downsync.objectSelectors expressions. When there are multiple matchLabels expressions, they are combined using a logical AND operation. Conversely, when there are multiple objectSelectors, they are combined using a logical OR operation."
+echo "-------------------------------------------------------------------------"
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v1alpha1
+kind: Placement
+metadata:
+  name: nginx-placement
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
```

Comment:
Need to update to latest placement with objects

Reply:
I think you're confusing labelSelectors with matchLabels?  Or maybe I am mistaken?